docs/web/using_with_svelte/translating.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: translating
+title: Translating with Tolgee for Svelte
+sidebar_label: Translating
+slug: /web/using_with_svelte/translating
+---
+When you have Tolgee libraries installed and TolgeeProvider is set up, you can start to translate your strings.
+There are several ways how to translate a text using Tolgee Svelte integration.
+
+## The T component
+
+First and the recommended way is the `<T>` component. You can use it like this:
+
+```svelte
+<script lang="ts">
+  import { T } from "@tolgee/svelte";
+</script>
+
+<T keyName="this_is_a_key" />
+```
+
+To provide parameters to you translation, use parameters prop.
+```svelte
+<T keyName="greeting" parameters={{name: "Josh"}} />
+```
+For translation `Hello {name}!` the rendered string will be `Hello Josh!`.
+
+Also, when you hold `Alt` key and move you mouse over the string, you should be able to click & translate the text
+in-context.
+
+We are using ICU message format. To read more about it, check
+[ICU Message Format](/docs/icu_message_format) documentation page.
+
+### Providing default value
+
+You can also provide default value as default slot of the `T` component. This value will be used when no translation
+have been found in localization data.
+```svelte
+<T keyName="this_is_a_key">This is default.</T>
+```
+
+## `getTranslate` method
+If you cannot use `T` component for some reason, then you can also use `getTranslate` method returning store containing
+a translation function. The usage is very simple, look:
+
+```svelte
+<script lang="ts">
```

Comment:
Don't use it with 'context="module"', as it is like this in svelte kit and it doesn't work.